dev_mode/package.json

```diff
@@ -22,7 +22,7 @@
     "@codemirror/view": "^6.9.6",
     "@jupyter/react-components": "^0.16.3",
     "@jupyter/web-components": "^0.16.3",
-    "@jupyter/ydoc": "^2.0.1",
+    "@jupyter/ydoc": "^3.0.0-a1",
```

Comment:
Small note, if we start using alpha here we should create an issue in jupyterlab that forces us to wait for stable 3.0.0 prior to 4.3.0 release.

Reply:
I opened https://github.com/jupyterlab/jupyterlab/issues/16649